packages/cli/src/commands/__tests__/build.test.js

```diff
@@ -1,5 +1,7 @@
-jest.mock('@redwoodjs/project-config', () => {
+vi.mock('@redwoodjs/project-config', async (importOriginal) => {
+  const mod = await importOriginal('@redwoodjs/project-config')
```

Comment:
In other places you've just done `await importOriginal()`. Here you pass in the name of the module you're importing. What's the difference?

Reply:
Nope, this one is a mistake, I'll will fix it.